Lombiq.UIKit/wwwroot/css/variables.css

```diff
@@ -0,0 +1,15 @@
+:root {
+    --input-text: black;
+    --input-bg: white;
+    --label-bg: white;
+    --label-bg-disabled: linear-gradient(180deg, white 40%, #f8f8f8 40%);
+    --input-border: black;
+}
+
+[data-theme="darkmode"], [data-bs-theme="dark"] {
+    --input-text: white;
+    --input-bg: #383838;
+    --label-bg: linear-gradient(180deg, #181a1b 40%, #383838 40%);
+    --label-bg-disabled: linear-gradient(180deg, #181a1b 40%, #222128 40%);
```

Comment:
I've changed the gradient value for disabled label because it looked extra cursed in dark mode:
![image](https://github.com/user-attachments/assets/32abecb8-cca7-42c2-865e-2f279a0b7e3d)

Reply:
This displeases Lucifer.